iptables/service.sls

```diff
@@ -36,9 +36,22 @@ iptables_{{ chain_name }}_policy:
     - table: filter
     - require_in:
       - iptables: iptables_flush
+
+iptables_{{ chain_name }}_ipv6_policy:
```

Comment:
It may be a good idea to check if host is IPv6 capable before trying to use ip6tables as it may fail on hosts where ipv6 is completely disabled (eg. because it's unwanted) or not available.